com.rclone.ui.app.yml

```diff
@@ -0,0 +1,61 @@
+id: com.rclone.ui.app
+
+runtime: org.gnome.Platform
+runtime-version: '46'
```

Comment:
```suggestion
runtime-version: '49'
```

Reply:
https://docs.flathub.org/docs/for-app-authors/requirements#end-of-life-dependencies